src/purchases.ts

```diff
@@ -666,6 +675,25 @@ export default class Purchases {
     RNPurchases.syncObserverModeAmazonPurchase(productID, receiptID, amazonUserID, isoCurrencyCode, price);
   }
 
+  /**
+   * Use this method only if you already have your own IAP implementation using StoreKit 2 and want to use
+   * RevenueCat's backend. If you are using StoreKit 1 for your implementation, you do not need this method.
+   *
+   * You only need to use this method with *new* purchases. Subscription updates are observed automatically.
+   *
+   * Important: This should only be used if you have enabled observer mode during SDK configuration.
```

Comment:
I think we don't do this in all methods... But we should document that this will fail if called on android or iOS <15